### PR TITLE
Add more sentry configuration params

### PR DESCRIPTION
**Pull request by alsamylkin**
It will let to specify params directly in a config, not thorough the code.

### CODE REVIEW

File: ulog/sentry/sentry.go [hunk 1]
```diff
@@ -44,6 +44,16 @@ var _zapToRavenMap = map[zap.Level]raven.Severity{
 	zap.FatalLevel: raven.FATAL,
 }
 
+// Configuration provides sentry DSN and other optional parameters for logging
+type Configuration struct {
+	DSN               string
+	MinLevel          *string                `yaml:"min_level,omitempty"`
+	TraceEnabled      *bool                  `yaml:"trace_enabled,omitempty"`
+	TraceSkipFrames   *int                   `yaml:"trace_skip_frames,omitempty"`
```
Comment (alsamylkin) [inline ulog/sentry/sentry.go]: Yep, make sense, thanks!

File: ulog/sentry/sentry_test.go [hunk 2]
```diff
@@ -158,6 +158,70 @@ func TestPacketSending(t *testing.T) {
...
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			conf, err := FromConfig(tc.conf)
+			if tc.isErr {
+				assert.Error(t, err)
+				return
+			}
+
+			assert.Nil(t, err)
```
Comment (sectioneight) [inline ulog/sentry/sentry_test.go]: require.NoError

File: ulog/sentry/sentry.go [hunk 3]
```diff
@@ -44,6 +44,18 @@ var _zapToRavenMap = map[zap.Level]raven.Severity{
 	zap.FatalLevel: raven.FATAL,
 }
 
+// Configuration provides sentry DSN and other optional parameters for logging
+type Configuration struct {
+	DSN      string
+	MinLevel *string                `yaml:"min_level,omitempty"`
```
Comment (sectioneight) [inline ulog/sentry/sentry.go]: why all the omit empty tags? iirc those are only useful if we serialize to YAML which we'd never do -- only deser for config

Comment (alsamylkin) [inline ulog/sentry/sentry.go]: Right, I've read the docs and assumed both ways, but tested and it is unnecessary for parsing.

File: ulog/sentry/sentry.go [hunk 4]
```diff
@@ -92,6 +104,43 @@ func New(dsn string, options ...Option) (*Hook, error) {
 	return sh, nil
 }
 
+// FromConfig returns a new Sentry Hook based on SentryConfiguration.
+func FromConfig(c Configuration) (*Hook, error) {
```
Comment (anuptalwalkar) [inline ulog/sentry/sentry.go]: nit: Make it less verbose, `sentry.Configure(c Configuration)`

File: ulog/sentry/sentry_test.go [hunk 5]
```diff
@@ -158,6 +159,70 @@ func TestPacketSending(t *testing.T) {
...
+			}, false},
+		{"ParseError", Configuration{MinLevel: &str}, nil, true},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			conf, err := FromConfig(tc.conf)
+			if tc.isErr {
+				assert.Error(t, err)
+				return
```
Comment (anuptalwalkar) [inline ulog/sentry/sentry_test.go]: nit: prefer else/

**Comment by madhuravi** [comment]
Fix ulog/doc.go, it currently has instructions to write code to disable traces. Change to use config the way you specified.

File: ulog/sentry/sentry_test.go (hunk 5 above)
Comment (alsamylkin) [inline ulog/sentry/sentry_test.go]: Just reducing nesting: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

**Pull request merged**